scripts/pylib/twister/twisterlib/harness.py

```diff
@@ -63,7 +62,7 @@ def __init__(self):
         self.run_id = None
         self.matched_run_id = False
         self.run_id_exists = False
-        self.instance = None
+        self.instance: TestInstance | None = None
```

Comment:
why add this decoration only?  this will mess the coding style.

Reply:
This is annotation to let user know, what kind of object is added to that class. I know, that we are not using annotations in twister, and I do not want to introduce any strong type checking, but rather I wanted to add annotation in couple of places to make that code more readable (cleaner, self-documented, improve code completion in IDE).